Composer/packages/client/src/utils/pageLinks.ts

```diff
@@ -42,17 +42,17 @@ export const topLinks = (
     {
       to: linkBase + `dialogs/${openedDialogId}`,
       iconName: 'SplitObject',
-      labelName: formatMessage('Design'),
+      labelName: formatMessage('Create'),
       disabled: !botLoaded,
       match: /(bot\/[0-9.]+)$|(bot\/[0-9.]+\/skill\/[0-9.]+)$/,
       isDisabledForPVA: false,
     },
     {
-      to: linkBase + `language-generation/${openedDialogId}`,
-      iconName: 'Robot',
-      labelName: formatMessage('Bot responses'),
-      disabled: !botLoaded || skillIsRemote,
-      match: /language-generation\/[a-zA-Z0-9_-]+$/,
+      to: `/bot/${rootProjectId || projectId}/botProjectsSettings`,
```

Comment:
Why aren't you using the link base?

Reply:
The project settings page only makes sense to be called from a root bot, not a skill. The rootProjectId, if it exists, is the ID of the root of a project, while "projectId" refers to the current bot we're looking at whether it's a root or a skill. It's a slightly clumsy workaround, but it works.